forms-flow-admin/src/components/users/users.tsx

```diff
@@ -372,22 +385,45 @@ const Users = React.memo((props: any) => {
       roles: selectedRolesIds,
       user: formData.user,
     };
-  CreateUser(
-    payload,
-    (data) => {
-      toast.success(t("User Added successfully!"));
-      clearForm();
-      closeInviteModal(); 
-      props.setInvalidated(true);
-    },
-    (err) => {
-      toast.error(t("User not exist!"));
-    }
-  );
+    CreateUser(
+      payload,
+      (data) => {
+        openSuccessModal();
+      },
+      (err) => {
+        setValidationError(t('User not exist!'));
```

Comment:
Please change message to "User doesn't exist!"

Reply:
Noted